### PR TITLE
Added instructions for toggle_key config parameter to Getting Started

### PR DESCRIPTION
**Pull request by 4lch4**
# Overview
As mentioned in #357, I went ahead and wrote up some instructions on how to use the `toggle_key` config parameter.

_**NOTE: All of my changes are confined to `get-started.md`.**_

## Minor Change/Formatting

I also cleaned up some whitespace at the end of lines and added language definitions for code blocks that were missing them. For example something like the following didn't have a language specified, so I added `bash` so it would resolve as a bash-esque syntax:

### BEFORE

\`\`\`
espanso status
\`\`\`

### AFTER

\`\`\`bash
espanso status
\`\`\`

Let me know if anything should be changed and I'd be happy to do so!

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -125,20 +128,63 @@ matches:
...
+```bash
 espanso restart
 ```
 
-Now try to type `:br` anywhere. If you did everything correctly, you should see `Best Regards` appear! 
+Now try to type `:br` anywhere. If you did everything correctly, you should see `Best Regards` appear!
+
+> In version 0.5.1, espanso introduced the `edit` subcommand which makes editing configuration files much easier. Take a look at [Quick Editing](/docs/configuration/#quick-editing) if you are interested.
+
+### Customizing the Toggle Key
```
Comment (federico-terzi) [inline docs/get-started.md:139]: Could we move this section below the `Useful shortcuts` section at the end?

Comment (4lch4) [reply docs/get-started.md]: There we go! I didn't even notice that at the bottom there, whoops 😅

**Comment by federico-terzi** [comment]
Thank you for the help @4lch4! Merged :)

**Comment by 4lch4** [comment]
> Thank you for the help @4lch4! Merged :)

Happy to help 😊

**Pull request merged**